code/__DEFINES/flags.dm

```diff
@@ -63,6 +63,7 @@
 #define NOGUNS			512
 #define NOBLOOD			1024
 #define NOFIRE			2048
+#define NOHUNGER		2049

```

Comment:
They aren't bitflags. If someone would merge https://github.com/tgstation/-tg-station/pull/17469 this would be obvious